pkgs/tools/misc/flashrom-stable/default.nix

```diff
@@ -30,7 +31,14 @@ stdenv.mkDerivation rec {
     libjaylink
     libusb1
   ] ++ lib.optionals (!stdenv.isDarwin) [
-    libgpiod
+    # Incompatible with libgpiod v2 API
+    (libgpiod.overrideAttrs (old: rec {
```

Comment:
How about adding a libgpiod16 or something in the top level, and use that as a depedency?

Reply:
Fixed.